### PR TITLE
Clean behat context constructors

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Remove all constructor in context, especially the ones accessing the container as it may occur befor it was initialized
| Type?             | bug fix
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green
| UI Tests          | ~
| Fixed issue or discussion?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by tleon — DISMISSED** [review]

File: tests/Integration/Behaviour/Features/Context/Domain/CmsPageFeatureContext.php
Comment (kpodemski) [inline tests/Integration/Behaviour/Features/Context/Domain/CmsPageFeatureContext.php]: @jolelievre typo

File: tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php
Comment (kpodemski) [inline tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php]: @jolelievre unused

Comment (jolelievre) [reply tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php]: Thanks I forgot to remove it, that was the initial point :D

File: tests/Integration/Behaviour/Features/Context/Domain/CmsPageFeatureContext.php
Comment (jolelievre) [reply tests/Integration/Behaviour/Features/Context/Domain/CmsPageFeatureContext.php]: Woups sorry, I didn't have time to re-read my own PR Thanks

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
